web_monitoring/db.py

```diff
@@ -515,21 +515,21 @@ def monitor_import_statuses(self, import_ids, stop=None):
         """
         Poll status of Version import jobs until all complete.
 
-        Use Ctrl+C to exit early. A list of the errors (so far) will be
-        returned.
+        Use Ctrl+C to exit early. A dict mapping the import IDs to any errors
+        from those imports (so far) will be returned.
 
         Parameters
         ----------
-        import_ids : collection
+        import_ids : iterable of (str or int)
```

Comment:
This is perfectly readable, and I'm not aware of a convention for this. There is a gradual migration to using the language of type hints here, which would express this as `Union[Iterable[str], Iterable[int]]`. I have mixed feelings about that movement; is gains something in specificity but loses something in readability to novices.

Reply:
Ah, ok. I’ll stick With this, then, especially since we aren’t actually using type annotations right now.